server/src/main/java/org/spine3/server/storage/memory/InMemoryCommandStorage.java

```diff
@@ -41,24 +41,22 @@
     @Override
     public void write(CommandId id, CommandStorageRecord record) {
         checkNotNull(id);
```

Comment:
How about having `checkNotDefault(M extends Message)`, which does non-null and non-default checks? We are going to have it over and over again.